tests/test_core.py

```diff
@@ -3731,7 +3730,7 @@ def test_dylink_printfs(self):
       void sidey() {
         printf("hello from side\n");
       }
-    ''', 'hello from main\nhello from side\n', force_c=True)
+    ''', 'hello from main\nhello from side\n', force_c=True, need_reverse=False)
```

Comment:
why disable the reverse? can this be with main module 1 instead?

(and either way please add a comment)

Reply:
Good catch.. reverted.